### PR TITLE
Add notification sound option

### PR DESCRIPTION
**Pull request by mervick**

### CODE REVIEW

**Comment by YurySolovyov** [comment]
Tests?

**Comment by mervick** [comment]
I will add tests soon

**Comment by mervick** [comment]
Added tests

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 extension/dist
+.idea
```
Comment (sindresorhus) [inline .gitignore]: This should be in your own global gitignore, not here.

File: extension/options.html [hunk 2]
```diff
@@ -40,9 +40,13 @@
...
- 			<h3>Desktop Notifications</h3>
- 			<input type="checkbox" id="show_desktop_notif">
- 			<label for="show_desktop_notif">Show desktop notifications</label>
- 		</section>
+			<h3>Notifications</h3>
+			<input type="checkbox" id="show_desktop_notif">
+			<label for="show_desktop_notif">Show desktop notifications</label>
+			<p>
+				<input type="checkbox" id="play_sound_notif">
+				<label for="play_sound_notif">Enable sound notifications</label>
```
Comment (sindresorhus) [inline extension/options.html]: > Enable notification sound

File: extension/options.html [hunk 3]
```diff
@@ -40,9 +40,13 @@
...
 		</section>
 		<section>
- 			<h3>Desktop Notifications</h3>
- 			<input type="checkbox" id="show_desktop_notif">
- 			<label for="show_desktop_notif">Show desktop notifications</label>
- 		</section>
+			<h3>Notifications</h3>
+			<input type="checkbox" id="show_desktop_notif">
+			<label for="show_desktop_notif">Show desktop notifications</label>
+			<p>
```
Comment (sindresorhus) [inline extension/options.html]: The above should also be placed in a `<p>` if this is.

File: extension/options.js [hunk 4]
```diff
@@ -67,6 +67,16 @@ document.addEventListener('DOMContentLoaded', () => {
 		}
 	});
 
+	const PlaySoundNotificationsOption = new Option({
+		id: 'play_sound_notif',
```
Comment (sindresorhus) [inline extension/options.js]: `play_notif_sound`

File: extension/options.js [hunk 5]
```diff
@@ -67,6 +67,16 @@ document.addEventListener('DOMContentLoaded', () => {
...
+		storageKey: 'playSoundNotif',
```
Comment (sindresorhus) [inline extension/options.js]: `playNotifSound`

File: extension/options.html [hunk 6]
```diff
@@ -40,9 +40,13 @@
...
+				<input type="checkbox" id="play_sound_notif">
```
Comment (sindresorhus) [inline extension/options.html]: Ideally, this one should only be enabled when `show_desktop_notif` is.

**Comment by mervick** [comment]
Fixed, except this one
> Ideally, this one should only be enabled when show_desktop_notif is.

As for me sound notifications plus counter is quite enough, I don't like pop-up notifications, and maybe for others will better to leave a choice too

**Pull request merged**